tests/field_defaults/tests.py

```diff
@@ -169,6 +171,17 @@ def test_bulk_create_mixed_db_defaults_function(self):
         years = DBDefaultsFunction.objects.values_list("year", flat=True)
         self.assertCountEqual(years, [2000, datetime.now().year])
 
+    def test_full_clean(self):
```

Comment:
Usually it's preferred to use an existing model so we don't have test fixture bloat :)